MiniGamesBox Inventory/src/main/java/plugily/projects/minigamesbox/inventory/normal/FastInv.java

```diff
@@ -1,398 +0,0 @@
-    this.itemHandlers.remove(slot);
-  }
-
-  /**
-   * Remove multiples {@link ItemStack} from the inventory.
-   *
-   * @param slots The slots where to remove the items
-   */
-  public void removeItems(int... slots) {
-    for(int slot : slots) {
-      removeItem(slot);
-    }
-  }
-
-  /**
-   * Add a close filter to prevent players from closing the inventory.
-   * To prevent a player from closing the inventory the predicate should return {@code true}.
-   *
-   * @param closeFilter The close filter
-   */
-  public void setCloseFilter(Predicate<Player> closeFilter) {
-    this.closeFilter = closeFilter;
-  }
-
-  /**
-   * Add a handler to handle inventory open.
-   *
-   * @param openHandler The handler to add.
-   */
-  public void addOpenHandler(Consumer<InventoryOpenEvent> openHandler) {
-    this.openHandlers.add(openHandler);
-  }
-
-  /**
-   * Add a handler to handle inventory close.
-   *
-   * @param closeHandler The handler to add
-   */
-  public void addCloseHandler(Consumer<InventoryCloseEvent> closeHandler) {
-    this.closeHandlers.add(closeHandler);
-  }
-
-  /**
-   * Add a handler to handle inventory click.
-   *
-   * @param clickHandler The handler to add.
-   */
-  public void addClickHandler(Consumer<InventoryClickEvent> clickHandler) {
-    this.clickHandlers.add(clickHandler);
-  }
-
-
-  /**
-   * Remove all handlers which handles inventory open.
-   */
-  public void removeOpenHandlers() {
-    this.openHandlers.clear();
-  }
-
-  /**
-   * Remove all handlers which handles inventory close.
-   */
-  public void removeCloseHandlers() {
-    this.closeHandlers.clear();
-  }
-
-  /**
-   * Remove all handlers which handles inventory click.
-   */
-  public void removeClickHandlers() {
-    this.clickHandlers.clear();
-  }
-
-
-  /**
-   * Open the inventory to a player.
-   *
-   * @param player The player to open the menu.
-   */
-  public void open(Player player) {
-    player.openInventory(this.inventory);
-  }
-
-  /**
-   * Get borders of the inventory. If the inventory size is under 27, all slots are returned.
-   *
-   * @return inventory borders
-   */
-  public int[] getBorders() {
-    int size = this.inventory.getSize();
-    return IntStream.range(0, size).filter(i -> size < 27 || i < 9 || i % 9 == 0 || (i - 8) % 9 == 0 || i > size - 9).toArray();
-  }
-
-  /**
-   * Get corners of the inventory.
-   *
-   * @return inventory corners
-   */
-  public int[] getCorners() {
-    int size = this.inventory.getSize();
-    return IntStream.range(0, size).filter(i -> i < 2 || (i > 6 && i < 10) || i == 17 || i == size - 18 || (i > size - 11 && i < size - 7) || i > size - 3).toArray();
-  }
-
-
-  /**
-   * Fill the inventory with items
-   *
-   * @param stack ItemStack to fill the inventory
-   */
-  public void fill(ItemStack stack) {
-    while(this.inventory.firstEmpty() != -1) {
-      setItem(this.inventory.firstEmpty(), stack, inventoryClickEvent -> inventoryClickEvent.setCancelled(true));
-    }
-  }
```

Comment:
Missing fill method

Reply:
We use `ItemMap` to set items as a map. This `fill` will require the inventory as an argument, and I don't want that as it's annoying.
In such a case, just use `setDefaultItem`.

Reply:
yea true with setDefaultItem all slots will be filed that are "empty" so it will do the same function, right.

Reply:
It's the same

Reply:
With a twist that you have to set it in `ItemMap` and call `refresh` after you change the map.

Reply:
The only FastInv method used internally is `setItem`. Others are just variants and will not work with this new library, unless you use the core `FastInv`